### PR TITLE
Seats availability aggregate, repository, tests

### PR DESCRIPTION
**Pull request by AlexanderLitus**
In progress.

### CODE REVIEW

File: registration/src/main/java/org/spine3/samples/lobby/registration/seat/availability/SeatsAvailabilityAggregate.java [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+import static com.google.common.base.Preconditions.checkState;
+import static com.google.common.collect.Lists.newLinkedList;
+import static org.spine3.samples.lobby.registration.util.CollectionUtils.findById;
+import static org.spine3.samples.lobby.registration.util.MessageFactory.newSeatQuantities;
+import static org.spine3.samples.lobby.registration.util.MessageFactory.newSeatQuantity;
+import static org.spine3.samples.lobby.registration.util.ValidationUtils.*;
+
+/**
+ * The aggregate root which manages the availability of conference seats.
```
Comment (alexander-yevsyukov) [inline registration/src/main/java/org/spine3/samples/lobby/registration/seat/availability/SeatsAvailabilityAggregate.java]: Just 'aggregate', without 'root'.

File: registration/src/main/java/org/spine3/samples/lobby/registration/seat/availability/SeatsAvailabilityAggregate.java [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+        final String reservationId = event.getReservationId().getUuid();
+        pendingReservations.remove(reservationId);
+
+        state.clearAvailableSeat();
+        state.addAllAvailableSeat(event.getAvailableSeatUpdatedList());
+
+        incrementState(state.build());
+    }
+
+    protected static class MakeSeatReservationCommandHandler {
```
Comment (alexander-yevsyukov) [inline registration/src/main/java/org/spine3/samples/lobby/registration/seat/availability/SeatsAvailabilityAggregate.java]: Please document.

File: registration/src/main/java/org/spine3/samples/lobby/registration/util/MessageFactory.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+import org.spine3.util.Identifiers;
+
+/**
+ * The utility class containing convenience methods for messages creation.
+ *
+ * @see Message
+ * @author Alexander Litus
+ */
+@SuppressWarnings("UtilityClass")
+public class MessageFactory {
```
Comment (alexander-yevsyukov) [inline registration/src/main/java/org/spine3/samples/lobby/registration/util/MessageFactory.java]: Please have utility classes specific for bounded contexts and groups of related classes.
Consider having utility class for common types, registration types, and money.

**Comment by AlexanderLitus** [comment]
@alexander-yevsyukov, PTAL.

**Pull request merged**